### ISSUE TITLE
`yield` syntax is experimental error with multiple features

### ISSUE THREAD

**Issue by Seeker14491** [intellij-rust/intellij-rust#3868]
This code shows no errors in the editor, as expected:

```rust
#![feature(generators)]

fn main() {
    let _generator = || {
        yield 1_u32;
    };
}
```

However, if you add more features next to the `generators` feature line, then an erroneous error is shown, as in the following:

```rust
#![feature(generators, box_syntax)] // <- another feature added to the same line

fn main() {
    let _generator = || {
        yield 1_u32; // `yield` syntax is experimental error
    };
}
```

## Environment

* **Intellij-Rust plugin version:** 0.2.97.2123-191
* **Rust toolchain version:** rustc 1.36.0-nightly (7d5aa4332 2019-05-16)
* **IDE name and version:** CLion 2019.1.2
* **Operating system:** Windows 10 x64
